src/wayland/CMakeLists.txt

```diff
@@ -36,4 +42,6 @@ configure_file(${CMAKE_CURRENT_SOURCE_DIR}/mirwayland.pc.in
 
 install(TARGETS     mirwayland                           LIBRARY DESTINATION "${CMAKE_INSTALL_FULL_LIBDIR}")
 install(FILES       ${CMAKE_CURRENT_BINARY_DIR}/mirwayland.pc    DESTINATION "${CMAKE_INSTALL_FULL_LIBDIR}/pkgconfig")
+install(DIRECTORY   ${CMAKE_SOURCE_DIR}/include/wayland/mir      DESTINATION "include/mirwayland")
 
+set(INCLUDEDIR "${CMAKE_INSTALL_PREFIX}/include/mirwayland")
```

Comment:
We need to add to the include file(s) to debian/libmirwayland-dev.install, or packaging will fail.

Reply:
Done